libs/utils/tracerAPI.ts

```diff
@@ -1,28 +1,46 @@
+import { constructCommitID } from '@context/UsersCommitContext/commitDispatch';
 import { ARBITRUM, ARBITRUM_RINKEBY, CommitTypeMap } from '@libs/constants';
 import { CommitEnum } from '@tracer-protocol/pools-js';
 
 export type SourceType = typeof ARBITRUM_RINKEBY | typeof ARBITRUM;
 
-const BASE_TRACER_API = process.env.NEXT_PUBLIC_TRACER_API;
+const query: (props: { pool?: string; account?: string; from?: number }) => string = ({ pool, account, from }) => `{
+    commits (where: { 
+        ${!!pool ? `pool: "${pool.toLowerCase()}",` : ''}
+        ${!!account ? `trader: "${account.toLowerCase()}",` : ''}
+        ${!!from ? `created_gt: ${from},` : ''}
```

Comment:
for pending commits you shouldn't need a from, just put `where: { isExecuted: false }`

Reply:
yep, had that orignally but didnt want to fix the sketchy front running thing, so just dropped it